src/content/docs/logs/ui-data/lookup-tables-ui.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Upload CSV-format lookup tables"
+metaDescription: 'In the New Relic logs UI, upload CSV-format lookup tables and use that data in combination with other New Relic data.'
+---
+
+import logsLookupTableUi from 'images/logs_screenshot-crop_lookup-table-ui.webp'
+
+Our **lookups** feature lets you enrich your log data, and other New Relic-stored data, with data about your business that you define in a CSV file. 
+
+## Why use lookups? [#overview]
+
+You can use lookups to make your telemetry data easier to understand, group data in custom ways, and include more information that wasn't available at the time for previously reported data. From the logs UI, you can upload a CSV file and use the `lookup()` function to include the lookup data in a NRQL query. 
+
+For reasons to use lookup tables, and how to use that data in queries, see [Query lookups](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups).
+
+<img
+  title="Upload a csv lookup table"
+  alt="Upload a csv lookup table"
+  src={logsLookupTableUi}
+/>
+
+## Requirements [#requirements]
+
+ In addition to [the standard requirements for a CSV file](https://www.rfc-editor.org/rfc/rfc4180), here are our requirements for CSV files: 
+
+* Each [account](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) has a limit of 20 lookup tables.
+* Each row must have the same number of columns as the header and there must be at least two columns.
+* Table names must conform to [the rules for event names](/docs/data-apis/ingest-apis/event-api/introduction-event-api/#limits), and they must:
+  * Start with an alphabetic character, and can contain alphanumeric characters, underscores (`_`) and colons (`:`) 
+  * Be less than 256 characters 
+* Do not use [reserved words](/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#reserved-words) for table names or column header values.
+* Max file size: 4 MB
+* Max row count: 20K
+
+Lookup table data can't be used in NRQL alert conditions. 
```

Comment:
This seems out-of-place here since it's a query limitation and not a requirement for CSV files.